src/PlaneWaveBasis.jl

```diff
@@ -119,10 +121,12 @@ build_kpoints(basis::PlaneWaveBasis, kcoords) =
     build_kpoints(basis.model, basis.fft_size, kcoords, basis.Ecut)
 
 # This is the "internal" constructor; the higher-level one below should be preferred
-@timing function PlaneWaveBasis(model::Model{T}, Ecut::Number,
+@timing function PlaneWaveBasis(model::Model{T},
+                                Ecut::Union{Number, Unitful.Quantity{T,D,Unitful.EnergyUnits}},
```

Comment:
That feels again a bit strongly typed in my opinion. How about just `Union{Number, Quantity}` (I suppose if you are `using Unitful` that `Quantity` is already in the main scope and thus you don't need the `Unitful.`, but check this).